release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -177,6 +177,16 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-dynamic-plugin-technology-preview"]
+==== Dynamic plug-in (Technology Preview)
+Starting with {product-title} 4.10, OpenShift console dynamic plug-ins are now available as a Technology Preview feature. This feature allows you make customizations to your interface at runtime such as: add custom pages, add perspectives and update navigation items, and add tabs and actions to resource pages. Plug-ins are registered with the console using the `ConsolePlugin` custom resource, and they are enabled in the console operator configuration by a cluster administrator.
```

Comment:
I might add a list here and try to make the first part more user-focused. (Saying that the program allows you to do something is a bit anthropomorphic.) Maybe something like:

```suggestion
Starting with {product-title} {product-version}, the ability to create OpenShift console dynamic plug-ins is now available as a Technology Preview feature. You can use this feature to customize your interface at runtime in many ways, including:

* Adding custom pages
* Adding perspectives and updating navigation items
* Adding tabs and actions to resource pages

After you register a plug-in with the console by using the `ConsolePlugin` custom resource, a cluster administrator must enable them in the console Operator configuration.
```

Are you creating multiple plug-ins? (I want to make sure we're describing the single feature correctly. It sounds like the feature is the ability to make a plug-in.)
Can anyone register the plug-in?

Reply:
You actually don't need admin privileges. Updating this sentence.

Reply:
You need to have authority to create a `ClusterPlugin` resource. By default, only cluster admins can create one.